Documentation/ContentObjects/Hmenu/Rootline.rst

```diff
@@ -26,95 +26,93 @@ and makes navigation to a certain page level easier.
     :ref:`Mount points <t3coreapi:MountPoints>` are supported.
 
 
-.. contents::
-   :local:
+..  contents::
+    :local:
 
+..  _hmenu-special-rootline-properties:
 
 Properties
 ==========
 
-.. _hmenu-special-rootline-range:
-.. ### BEGIN~OF~TABLE ###
+..  _hmenu-special-rootline-range:
 
-.. container:: table-row
+special.range
+-------------
 
-   Property
-         special.range
+..  confval:: special.range
+    :name: hmenu-rootline-special-range
+    :type: string /:ref:`stdWrap <stdwrap>`
+    :Syntax: `[begin-level] | [end-level]`
 
-   Data type
-         string /:ref:`stdWrap <stdwrap>`
+    `[begin-level]` and `[end-level]` are counted like the
```

Comment:
I would leave the "|" here as it is part of the syntax. Additionally, "counted like" seems not to have the same meaning as in the original sentence.